src/software/embedded/ansible/tasks/setup_systemd.yml

```
@@ -16,6 +16,10 @@
         dest: ~/thunderbots_binaries/
         copy_links: yes
 
+    - name: Computer Thunderloop MD5 Hash
```

I think this is a typo?
```suggestion
    - name: Compute Thunderloop MD5 Hash
```

---

Oops, yes